_posts/2016-12-28-Day_Twentyone.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: 8th Light apprenticeship - Day Twenty-one | Demystifying Elixir macros
+categories: 8thLight apprenticeship
+---
+
+A disclaimer before we begin: I am not advocating the use of macros. Quite the
+opposite in fact, as it will hopefully be clear throughout the article. Deciding 
```

Comment:
not sure if you make it clear throughout your blogpost that you advocate avoiding macros, other than the fact that you just stated it here.
It's not the purpose of the blogpost to explain why macros should be avoided, but since you do make reference to this subject, might be worth having a sentence on why one might avoid using them. This could be added wither at the beginning or the end of the post